requirements/common_constraints.txt

```diff
@@ -23,3 +23,5 @@ Django<3.3
 # elasticsearch>=7.14.0 includes breaking changes in it which caused issues in discovery upgrade process.
 # elastic search changelog: https://www.elastic.co/guide/en/enterprise-search/master/release-notes-7.14.0.html
 elasticsearch<7.14.0
+
+setuptools<60
```

Comment:
Can you provide a reason here in comments as why you pinned the setuptools?

Reply:
This was part of the automated update - I'll double check

Reply:
Yep it was added here: https://github.com/openedx/edx-lint/commit/d3a2654d55f30839a81cc796264e25a8fb7464d0